common/models/order-config-model.js

```diff
@@ -12,7 +12,7 @@ module.exports = function (OrderConfigModel) {
         OrderConfigModel.fetchFileImportHeaders = function (id, options) {
             var importableHeaders = {
                 Inventory: ['orderQuantity', 'fulfilledQuantity'],
-                Product: ['name', 'sku'],
+                Product: ['name', 'sku','supplyprice'],
```

Comment:
rename `supplyprice` to be in camelCase similar to other keys 

And maybe move them to an existing constants file

Reply:
done !